storage-bigtable/src/lib.rs

```diff
@@ -364,11 +364,19 @@ impl From<LegacyTransactionByAddrInfo> for TransactionByAddrInfo {
 
 pub const DEFAULT_INSTANCE_NAME: &str = "solana-ledger";
 
+// stringified = truev, alue is a stringified credential
+// stringified = false, value is a filepath (default)
+#[derive(Debug)]
+pub struct CredentialOption {
+    pub stringified: bool,
+    pub value: String,
+}
```

Comment:
I think I still like this better as an enum, so that the type of `None` is more explicit:
```suggestion
pub enum CredentialType {
    Filepath(Option<String>),
    Stringified(String),
}
```